proposals/automated-go-ipfs-releases.md

```diff
@@ -0,0 +1,121 @@
+
+#### User workflow example
+_How would a developer or user use this new capability?_
+<!--(short paragraph)-->
+
+A go-ipfs maintainer could run a script or GitHub action that would automatically perform a release from a given branch instead of manually performing the release.
+
+A go-ipfs user would be able to set an option in their config file such that on boot they would be able to automatically update the software they were running. Additionally, package managers with restrictive environments could just distribute shims around go-ipfs and the shim would be able to update and maintain itself.
+
+#### Impact
+_How would this directly contribute to web3 dev stack product-market fit?_
+
+<!--
+Explain how this addresses known challenges or opportunities.
+What awesome potential impact/outcomes/results will we see if we nail this project?
+-->
+
+Users having up to date versions of go-ipfs will give them a better experience resulting in fewer complaints about long solved issues, lower ecosystem burden and an easier time updating protocols.
+
+#### Leverage
+_How much would nailing this project improve our knowledge and ability to execute future projects?_
+
+<!--
+Explain the opportunity or leverage point for our subsequent velocity/impact (e.g. by speeding up development, enabling more contributors, etc)
+-->
+
+Future projects would make it to users faster and there would be less time from developers distracted by release process.
+
+#### Confidence
+_How sure are we that this impact would be realized? Label from [this scale](https://medium.com/@nimay/inside-product-introduction-to-feature-priority-using-ice-impact-confidence-ease-and-gist-5180434e5b15)_.
+
+<!--Explain why this rating-->
+
+
+## Project definition
+#### Brief plan of attack
+
+<!--Briefly describe the milestones/steps/work needed for this project-->
+
+#### What does done look like?
+_What specific deliverables should completed to consider this project done?_
+
+It takes less than 1hr to do a go-ipfs release from when the release is ready to be tagged, and users can automatically update to the latest version of go-ipfs.
+
+####  What does success look like?
+_Success means impact. How will we know we did the right thing?_
+
+<!--
+Provide success criteria. These might include particular metrics, desired changes in the types of bug reports being filed, desired changes in qualitative user feedback (measured via surveys, etc), etc.
+-->
+
+#### Counterpoints &amp; pre-mortem
+_Why might this project be lower impact than expected? How could this project fail to complete, or fail to be successful?_
+
+- There are external factors slowing down releases other than the process itself
+- It takes too much effort to automate the release process
```

Comment:
If we have an idea on the current and future state plus an estimate of how much work this will take, we can assess the return on investment.  For example, estimated data that would be help:
1. Amount of dev effort this process takes today
2. Amount of dev effort the process will take after doing this project
3. Estimated effort to do this project
4. Number of releases we do in a year